torch/csrc/jit/python/script_init.cpp

```diff
@@ -898,19 +898,25 @@ void initJitScriptBindings(PyObject* module) {
           "_save_for_mobile",
           [](Module& m,
              const std::string& filename,
-             const ExtraFilesMap& _extra_files = ExtraFilesMap()) {
-            m._save_for_mobile(filename, _extra_files);
+             const ExtraFilesMap& _extra_files = ExtraFilesMap(),
+             bool _save_debug_info_in_bytecode = false) {
+            m._save_for_mobile(
+                filename, _extra_files, _save_debug_info_in_bytecode);
           },
           py::arg("filename"),
-          py::arg("_extra_files") = ExtraFilesMap())
+          py::arg("_extra_files") = ExtraFilesMap(),
+          py::arg("_save_debug_info_in_bytecode") = false)
```

Comment:
Is this covered by tests from Python side? The tests could be added in test/mobile/test_lite_script_module.py.

Reply:
I added Python tests. Since we have no access to the internal C++ object, I basically check if the outputs still match when we enable debug info saving and if module info strings are contained in the debug info buffer. Let me know if that suffices.